tailwind.purge.safelist.txt

```diff
@@ -60,3 +60,7 @@ tw-text-subhead
 tw-text-base
 tw-text-small
 tw-text-link
+
+# Whitespace Classes
```

Comment:
In the end I was getting some flaky behavior so I decided to add these to the purge list